tools/meta/templates/processor.cpp

```diff
@@ -42,7 +42,7 @@ const ProcessorInfo {{ file/name }}::processorInfo_{
 
 const ProcessorInfo {{ file/name }}::getProcessorInfo() const { return processorInfo_; }
 
-{{ file/name }}::{{file / name}}()
+{{ file/name }}::{{ file/name}}()
```

Comment:
Should it start with a space?

```suggestion
{{ file/name }}::{{file/name}}()
```

Reply:
generally we have a space before and after